lib/components/navbar/NavLink.tsx

```diff
@@ -0,0 +1,16 @@
+import Link from "next/link";
+import { withRouter } from "next/router";
+
+const NavLink = props => (
+  <Link href={props.path}>
+    <div
+      className={`NavButton ${
+        props.router.pathname === props.path ? "active" : ""
```

Comment:
📌 This is interesting, although we will eventually want to replace this with Chakra UI components